admin/templates/pre-reg/prereg.html

```diff
@@ -0,0 +1,70 @@
+{% extends 'base.html' %}
+{% load render_bundle from webpack_loader %}
+
+{% block content %}
+{% render_bundle 'vendor' %}
```

Comment:
There's a block called `top_includes` that appears right before the closing head tag. All pre-content javascript and styles should probably go in there. Also, render_bundle 'vendor' is already being called in base.html at the bottom of the page.